docs/measuring-experiments/adding-screenshots.md

```diff
@@ -1 +1,12 @@
 # Adding Screenshots
+In your experiments, you can add screenshots to your different variations. You can use these screenshots to highlight the changes made in the variation.
```

Comment:
Can you make it clear in these pages, and any following ones, that this all requires first navigating to the "experiments" tab on the left hand side. Just so they know where to go